hack/addons/update-aws-ebs-csi.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly SCRIPT_DIR
+
+# shellcheck source=hack/common.sh
+source "${SCRIPT_DIR}/../common.sh"
+
+if [ -z "${AWS_EBS_CSI_VERSION:-}" ]; then
+  echo "Missing environment variable: AWS_EBS_CSI_VERSION"
+  exit 1
+fi
+
+readonly ASSETS_DIR="${GIT_REPO_ROOT}/.local/csi/aws-ebs/${AWS_EBS_CSI_VERSION}"
+mkdir -p "${ASSETS_DIR}"
+
+FILENAME="aws-ebs-csi.yaml"
+
+kubectl create -k \
+  "github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=${AWS_EBS_CSI_VERSION}" \
+  --dry-run=client -o yaml |
+  sed 's|imagePullPolicy: Always|imagePullPolicy: IfNotPresent|g' \
+    >"${ASSETS_DIR}/${FILENAME}"
+
+# Add default storage class here.
+cat <<EOF >"${ASSETS_DIR}/aws-ebs-csi-temp.yaml"
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: ebs-sc
```

Comment:
Should this match the csi provider name?
```suggestion
  name: aws-ebs
```